package.json

```diff
@@ -9,10 +9,16 @@
   ],
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
   "scripts": {
     "test": "ts-node node_modules/jasmine/bin/jasmine.js",
     "test:coverage": "ts-node node_modules/istanbul/lib/cli.js cover -e .ts  -x \"*.d.ts\" -x \"*.spec.ts\" node_modules/jasmine/bin/jasmine.js",
-    "build": "tsc"
+    "build": "tsc",
+    "preversion": "npm test",
+    "postversion": "git push && git push --tags",
```

Comment:
Ah, I've just published a new version and this obviously pushes to my fork (`origin`) instead of the main repo (`upstream`). So in general this should be `git push upstream && git push upstream --tags`, but it would break if someone names it differently or work on this repo directly. The only solid solution I can think of would be using an absolute URL. Ideas/thoughts?

Reply:
Oh additionally, I cannot directly push to the repo (which is fine, apart from version update commits/tags):
```
jonatanklosko@mockingjay:~/projects/wca-helpers (master) $ git push upstream master
Counting objects: 4, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 381 bytes | 381.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0)
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: Required status check "continuous-integration/travis-ci" is expected. At least 1 approving review is required by reviewers with write access.
To github.com:thewca/wca-helpers.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:thewca/wca-helpers.git'
```